README.md

```diff
@@ -1,2 +1,2 @@
-# di-one-login-home-stubs
-Repository to store stubs for services that OneLogin Home integrates with
+# di-account-management-stubs
+Repository to store stubs for services that Gov.UK OneLogin Account Managemet integrates with
```

Comment:
```suggestion
Repository to store stubs for services that GOV.UK One Login Account Management integrates with
```
Super nitpick - it's always GOV.UK in capitals and our product's called One Login. We're not very good at being consistent but I'm trying to correct it where I spot it